### PR TITLE
Update API Tokens documentation

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/reference/admin/api_tokens/index.md [hunk 1]
```diff
@@ -9,24 +9,25 @@ weight: 20030
 
 ## Managing API Tokens
 
-To access the API Tokens administration, click on the {{< icon key >}} icon in the navigation bar.
```
Comment (ccedric) [inline content/reference/admin/api_tokens/index.md:12]: Why did you remove this line?

File: content/reference/admin/api_tokens/index.md [hunk 2]
```diff
@@ -9,24 +9,25 @@ weight: 20030
...
-
 {{< img src="tokens.png" alt="Tokens" >}}
 
-To create an API token, click on the **Create** button. Once the API token is created, copy the token, as you won't be able to retrieve it again.
+To create an API token, click on the **Create** button.
+Once the API token is created, copy the token, as you won't be able to retrieve it again.
 
 {{< img src="create-token.png" alt="Create token" >}}
 
 There are three permissions available for an API Token:
```
Comment (guilgaly) [inline content/reference/admin/api_tokens/index.md:19]:
```suggestion
There are four permission levels available for API Tokens:
```

File: content/reference/admin/api_tokens/index.md (hunk 1 above)
Comment (tpetillot) [reply content/reference/admin/api_tokens/index.md:12]: We don't have the needed icons anymore. And I don't see the point of saying "To access API Tokens, click on API tokens"...

File: content/reference/admin/api_tokens/index.md [hunk 3]
```diff
@@ -9,24 +9,25 @@ weight: 20030
 
 ## Managing API Tokens
 
-To access the API Tokens administration, click on the {{< icon key >}} icon in the navigation bar.
-
 {{< img src="tokens.png" alt="Tokens" >}}
 
-To create an API token, click on the **Create** button. Once the API token is created, copy the token, as you won't be able to retrieve it again.
+To create an API token, click on the **Create** button.
+Once the API token is created, copy the token, as you won't be able to retrieve it again.
```
Comment (guilgaly) [inline content/reference/admin/api_tokens/index.md:15]:
```suggestion
Once the API token is created, make sure to copy the token, as you won't be able to retrieve it again later.
```

File: content/reference/admin/api_tokens/index.md [hunk 4]
```diff
@@ -9,24 +9,25 @@ weight: 20030
...
 
 {{< img src="create-token.png" alt="Create token" >}}
 
-There are three permissions available for an API Token:
+There are four permissions available for API Tokens:
 
-- The Start permission, allowing to start simulations
 - The Read permission, allowing to read all the data from runs (typically useful in Grafana)
+- The Start permission, allowing to start simulations
```
Comment (guilgaly) [inline content/reference/admin/api_tokens/index.md:22]:
```suggestion
- The Start permission, allowing to start simulation runs
```

File: content/reference/admin/api_tokens/index.md [hunk 5]
```diff
@@ -9,24 +9,25 @@ weight: 20030
...
 - The Artifacts permission, allowing to upload artifacts
 - The All permission, combining all three of the previous permissions
 
+You can set a permission globally or to a specific team.
```
Comment (guilgaly) [inline content/reference/admin/api_tokens/index.md:26]:
```suggestion
You can set a permission globally or within a specific team only.
```

File: content/reference/admin/api_tokens/index.md (hunk 1 above)
Comment (ccedric) [reply content/reference/admin/api_tokens/index.md:12]: You should at least say where the user needs to click to open the page, even if you don't display the icon.

**Review by Isammoc — APPROVED** [review]
